boundingbox.go

```diff
@@ -26,3 +29,128 @@ func decodeBoundingBox(bb interface{}) ([]float64, error) {
 		return nil, fmt.Errorf("bounding box property not usable, got %T", bb)
 	}
 }
+
+func checkBoundingBox(min, max [2]float64) bool {
+	return min[0] <= max[0] && min[1] <= max[1]
+}
+
+// GetBoundingBox  Get Geometry geo outsourcing box
+func (g *Geometry) GetBoundingBox() (min, max [2]float64, ok bool) {
```

Comment:
I would suggest changing these methods to `BoundingBox() (min, max [2]float64)`. Go tends to dislike "Get*" method names.  I'd also put avoid a bool, just leave that up to the caller, zero values would be invalid.

Reply:
Yes, what you said makes sense. I will take your suggestions and make changes.